src/preferences/effectchainpresetlistmodel.cpp

```diff
@@ -0,0 +1,71 @@
+#include "preferences/effectchainpresetlistmodel.h"
+
+#include <QMimeData>
+
+namespace {
+const QString kMimeTextDelimeter = "\n";
```

Comment:
```suggestion
const QString kMimeTextDelimeter = QStringLiteral("\n");
```

Reply:
fixed in ec1146e43afc9ba6e0e998a85844301da9ec20f5